components/mpas-ocean/src/shared/mpas_ocn_forcing.F

```diff
@@ -180,6 +182,48 @@ subroutine ocn_forcing_build_fraction_absorbed_array(meshPool, statePool, forcin
            end do
         end do
 
+!  now do subglacial runoff separately
+        if ( trim(config_subglacial_runoff_mode) == 'data' ) then
+            if ( trim(config_sgr_flux_vertical_location) == 'top' ) then
```

Comment:
Would it make sense to add and `else` this this and raise an error if `config_sgr_flux_vertical_location` has an unexpected value?  Otherwise, I worry that a typo could run fine but have weird and hard-to-explain behavior.

Reply:
What is an appropriate way to do this? to raise error and kill the run?

Reply:
Done